fdbrpc/include/fdbrpc/TokenSign.h

```diff
@@ -90,9 +90,9 @@ struct TokenRef {
 	Optional<StringRef> issuer; // iss
 	Optional<StringRef> subject; // sub
 	Optional<VectorRef<StringRef>> audience; // aud
-	Optional<uint64_t> issuedAtUnixTime; // iat
-	Optional<uint64_t> expiresAtUnixTime; // exp
-	Optional<uint64_t> notBeforeUnixTime; // nbf
+	Optional<double> issuedAtUnixTime; // iat
+	Optional<double> expiresAtUnixTime; // exp
+	Optional<double> notBeforeUnixTime; // nbf
```

Comment:
I don't think these should be double. Int is easier (as we don't have to worry about any floating point weirdness) and given the expected TTL of tokens and clock screw across machines, anything more precise than a second imho doesn't make a lot of sense